app/services/preserved_object_handler.rb

```diff
@@ -42,7 +42,7 @@ def create_after_validation
       create_db_objects(PreservedCopy::INVALID_MOAB_STATUS, true)
     end
 
-    handler_results.log_results
+    handler_results.report_results
     handler_results.result_array
```

Comment:
if `report_results` method returned the `result_array`, then we could end pohandler public methods with:

```
handler_results.report_results
```

instead of 
```
handler_results.report_results
handler_results.result_array
```